Emby.Server.Implementations/Session/SessionManager.cs

```diff
@@ -1685,48 +1709,53 @@ private string GetImageCacheTag(BaseItem item, ImageType type)
             }
         }
 
+        /// <inheritdoc />
         public void ReportNowViewingItem(string sessionId, string itemId)
         {
             if (string.IsNullOrEmpty(itemId))
             {
                 throw new ArgumentNullException(nameof(itemId));
             }
 
-            //var item = _libraryManager.GetItemById(new Guid(itemId));
+            var item = _libraryManager.GetItemById(new Guid(itemId));
 
-            //var info = GetItemInfo(item, null, null);
+            var info = GetItemInfo(item, null);
 
-            //ReportNowViewingItem(sessionId, info);
+            ReportNowViewingItem(sessionId, info);
```

Comment:
this doesn't look like fixing warnings

Reply:
The method isn’t used anywhere, and the commented out code was causing warnings because they don’t start with a space